Sumo Orgs Flex docs (#4962)
SumoLogic/sumologic-documentation: blog-service/2025-01-16-manage.md

---

> title: Create and Manage Orgs:Flex (Manage)

Reviewer: ```suggestion
title: Introducing Sumo Logic Organizations for Flex Customers (Manage)
```

image: https://help.sumologic.com/img/sumo-square.png
keywords:
  - manage

>   - create-manage-orgs-flex

Reviewer: ```suggestion
  - create-manage-orgs-flex
  - flex-plan
```

hide_table_of_contents: true    
---

import useBaseUrl from '@docusaurus/useBaseUrl';

<a href="https://help.sumologic.com/release-notes-service/rss.xml"><img src={useBaseUrl('img/release-notes/rss-orange2.png')} alt="icon" width="50"/></a>

> We are excited to announce our Sumo Logic's Organizations ("Sumo Orgs") feature for Sumo Logic Flex customers. With this release, Flex users can effectively group, provision, manage, and monitor the credit usage across multiple organizations. [Learn more](/docs/manage/manage-subscription/create-manage-orgs-flex).

Reviewer: ```suggestion
We are excited to announce our Sumo Logic's Organizations ("Sumo Orgs") feature for Sumo Logic Flex customers. With this release, Flex users can effectively group, provision, manage, and monitor credit usage across multiple organizations, providing greater visibility and control over account structures. [Learn more](/docs/manage/manage-subscription/create-manage-orgs-flex).
```
